template/build/webpack.dev.conf.js

```diff
@@ -31,6 +31,7 @@ const devWebpackConfig = merge(baseWebpackConfig, {
     hot: true,
     contentBase: false, // since we use CopyWebpackPlugin.
     compress: true,
+    https: config.dev.https,
```

Comment:
Thanks! We still have to add the actual property to `/config/index.js` file so people can know about the option's existance without studying the webpack files.